README.md

```diff
@@ -671,7 +693,41 @@ If Ddosify can't receive the response for a request, that step is marked as Fail
 | `(status_code == 200) \|\| (status_code == 201)`   | same as preceding one |
 | `regexp(body,\"[a-z]+_[0-9]+\",0) == \"messi_10\"`   | checks if matched result from regex is equal to "messi_10" |
 
+## Test-Wide Assertion
+
+Ddosify now supports test-wide assertions, allowing users to verify the success of their load tests based on response times and failure counts of iterations. With this feature, users can assert the percentile of response times and the failure counts of all the iterations in a test.
```

Comment:
- Ddosify now supports -> Let's remove _**now**_

- Let's put the below paragraph at the end of this section.

**Difference Between Test-Wide Assertions and Step Assertions**
Unlike assertions focused on individual steps, which determine the success or failure of a step according to its response, Test-Wide assertions create an abort/continue logic for the entire test, which is based on the accumulated data from all iterations.